extraction/utils/extraction_utils.py

```diff
@@ -0,0 +1,9 @@
+"""This script contains helper utilities for extractors"""
+
+
+def get_size(node):
```

Comment:
name to get_video_size. Probably name the file to video_utils

(Will this method be used in multiple files, you could just make it part of video_extractor.py - upto you)

Reply:
I just saw that the image PR also a get_size method - is it using the same method? Probably you could renamt to get_media_size and the file to media_extraction_utils.py

Reply:
Added!